docs/source/tutorial_worlds.md

```diff
@@ -1,9 +1,9 @@
-Data Handling and Batching
+Worlds, Sharing & Batching
 ==========================
 
-__Authors__: Alexander Holden Miller, Kurt Shuster
+__Authors__: Alexander Holden Miller, Kurt Shuster, Stephen Roller
 
-:::{tip} Before you begin
+:::{important} Before you begin
```

Comment:
this doesn't work  in the github "rich diff" mode but maybe it isn't supposed to? hard to see if it works tho from here in that case..

Reply:
it works